homeassistant/components/logbook.py

```diff
@@ -115,6 +115,68 @@ def log_message(service):
     return True
 
 
+class DomainsAndEntitiesFilter:
```

Comment:
we should use `helpers/entityfilter.py`

Reply:
Looked into `entityfilter.py`. Adding it as a filter, surfaces inconsistency: Basically this test starts failing: https://github.com/home-assistant/home-assistant/blob/dev/tests/components/test_logbook.py#L334 
It expects both `switch.bla` events to be filtered out (because those are specified under exclude domains list). While `entityfilter` spcifically says that in this situation exclude domain list is ignored: https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/helpers/entityfilter.py#L78

Documentation seems unclean in that case as well to me. Not sure what is actually right, and if its ok to introduce backwards incompatible change here?

Reply:
The reason exclude domains are ignored, is because if you're explicit about what to include, you just need to make sure you don't exclude any domains that you also include?

Reply:
it sounds good to me, i just wonder if that would be ok if i will update the test. that would be backwards incompatible change, that we should announce somehow?